### ISSUE TITLE
precision recall and confusion matrix

### ISSUE THREAD

**Issue by strawhatrag** [strawhatrag/demo#1]
![Screenshot 2024-03-14 215747](https://github.com/strawhatrag/demo/assets/77497081/af49dfd4-1e69-4391-9add-88efa3f5e1cf)

## output in text:

7/7 [==============================] - 2s 174ms/step - loss: 12.8490 - accuracy: 0.5136 - val_loss: 0.6140 - val_accuracy: 0.8036
Epoch 2/10
7/7 [==============================] - 1s 143ms/step - loss: 3.8401 - accuracy: 0.6409 - val_loss: 1.9731 - val_accuracy: 0.6786
Epoch 3/10
7/7 [==============================] - 1s 143ms/step - loss: 0.9910 - accuracy: 0.7909 - val_loss: 0.6146 - val_accuracy: 0.8929
Epoch 4/10
7/7 [==============================] - 1s 146ms/step - loss: 0.6202 - accuracy: 0.8909 - val_loss: 0.5992 - val_accuracy: 0.9107
Epoch 5/10
7/7 [==============================] - 1s 146ms/step - loss: 0.3381 - accuracy: 0.9455 - val_loss: 0.8066 - val_accuracy: 0.9107
Epoch 6/10
7/7 [==============================] - 1s 147ms/step - loss: 0.1799 - accuracy: 0.9500 - val_loss: 0.6108 - val_accuracy: 0.9107
Epoch 7/10
7/7 [==============================] - 1s 147ms/step - loss: 0.1615 - accuracy: 0.9455 - val_loss: 0.5978 - val_accuracy: 0.9107
Epoch 8/10
7/7 [==============================] - 1s 146ms/step - loss: 0.1131 - accuracy: 0.9773 - val_loss: 0.4358 - val_accuracy: 0.9286
Epoch 9/10
7/7 [==============================] - 1s 150ms/step - loss: 0.1053 - accuracy: 0.9773 - val_loss: 0.3640 - val_accuracy: 0.9286
Epoch 10/10
7/7 [==============================] - 1s 152ms/step - loss: 0.0581 - accuracy: 0.9909 - val_loss: 0.1882 - val_accuracy: 0.9107
C:\Users\nandana\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\keras\src\engine\training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
2/2 [==============================] - 0s 27ms/step - loss: 0.1882 - accuracy: 0.9107
Test loss: 0.1882367581129074, Test accuracy: 0.9107142686843872
2/2 [==============================] - 0s 25ms/step
Confusion Matrix:
[[22  4]
 [ 1 29]]
Precision: 0.8787878787878788
Recall: 0.9666666666666667

Results:
--------------
| Metric     | Value   |
|------------|---------|
| Accuracy   | 0.9107 |
| Precision  | 0.8788 |
| Recall     | 0.9667 |
--------------
